docs/Fields.md

```diff
@@ -58,219 +44,380 @@ const PostShow = ({ id }) => {
+    </Edit>
+);
 ```
 
-**Tip**: If you want to format a field according to the value, use a higher-order component to do conditional formatting, as described in the [Theming documentation](./Theming.md#conditional-formatting).
+React-admin comes with about 20 field components, specialized in rendering numbers, image URLs, booleans, arrays, etc. And if you can't find a field for your need, you can always create your own.
 
-**Tip**: If your interface has to support multiple languages, don't use the `label` prop, and put the localized labels in a dictionary instead. See the [Translation documentation](./Translation.md#translating-resource-and-field-names) for details.
+## Common Field Props
 
-## Recipes
+All Field components accept the following props:
 
-### Styling Fields
+| Prop                          | Required | Type                           | Default  | Description                                                                                                                                             |
+|-------------------------------| -------- |--------------------------------| -------- |---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [`source`](#source)           | Required | `string`                       | -        | Name of the property to display                                                                                                                         |
+| [`label`](#label)             | Optional | `string` &#124; `ReactElement` | `source` | Used as a table header or an input label                                                                                                                |
+| [`record`](#record)           | Optional | `Object`                       | -        | Object containing the properties to display, to override the record from the current `RecordContext`                                                    |
+| [`sortable`](#sortable)       | Optional | `boolean`                      | `true`   | When used in a `List`, should the list be sortable using the `source` attribute? Setting it to `false` disables the click handler on the column header. |
+| [`sortBy`](#sortby)           | Optional | `string`                       | `source` | When used in a `List`, specifies the actual `source` to be used for sorting when the user clicks the column header                                      |
+| [`sortByOrder`](#sortbyorder) | Optional | `ASC` &#124; `DESC`            | `ASC`    | When used in a `List`, specifies the sort order to be used for sorting when the user clicks the column header                                           |
+| [`className`](#classname)     | Optional | `string`                       | -        | A class name (usually generated by JSS) to customize the look and feel of the field element itself                                                      |
```

Comment:
I think we're missing `headerClassName` and `cellClassName`. Did we remove them already?

Reply:
We didn't remove them, but I chose not to document them anymore. With the `sx` prop, they are no longer the best way to style a column. It's documented in Datagrid.md

Reply:
I missed the part referencing a field :+1: